### PR TITLE
Added CI.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by bwplotka** [comment]
cc @iNecas

File: .github/workflows/go.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: go
+
+on:
+  push:
+    branches:
+      - master
```
Comment (iNecas) [inline .github/workflows/go.yaml:6]: Given the latest movements about the naming I wonder if not to start with `main` right away to save some potential work later on :) Just an idea

**Comment by bwplotka** [comment]
Apparently GitHub is working on some quick option to do that for everyone if wanted, so let's wait on that! WDYT? (:

**Comment by bwplotka** [comment]
Lint is failing for a good reason:

> ##[error]/home/runner/work/obslytics/obslytics/pkg/input/factory/factory.go:4:2: package "errors" shouldn't be imported, suggested: "github.com/pkg/errors"
##[error]/home/runner/work/obslytics/obslytics/pkg/output/factory/factory.go:4:2: package "errors" shouldn't be imported, suggested: "github.com/pkg/errors"
make: *** [lint] Error 3

Let's fix in separate PR (:

**Pull request merged**